compiler_opt/rl/data_collector.py

```diff
@@ -126,7 +127,7 @@ def wait(self, get_num_finished_work):
 
     Args:
       get_num_finished_work: a callable object which returns the amount of
-      finished work.
+        finished work.
```

Comment:
spurious?

Reply:
unrelated to the change but my python autoformatter prefers it this way. consistent with the file as well: see the docstring for thresholds on the EarlyExitChecker __init__ method.